src/compiler/crystal/syntax/to_s.cr

```diff
@@ -1414,12 +1414,11 @@ module Crystal
           printed_arg = true
         end
         if named_args = node.named_args
-          @str << ", " if printed_arg
-          named_args.each do |named_arg|
+          named_args.each_with_index do |named_arg, i|
+            @str << ", " if printed_arg || i > 0
```

Comment:
Again you're right ... So much changes for such a small modification, sorry for that and thank for your time :)